lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -96,7 +96,7 @@
     description:
       - Path to a file containing environment variables I(FOO=BAR).
       - If variable also present in C(env), then C(env) value will override.
-      - Requires docker >= 2.3.0.
+      - Requires docker-py >= 1.4.0.
```

Comment:
Align with `MIN_DOCKER_VERSION`

Reply:
I removed it, since the general requirements for the module already specify `docker-py >= 1.7.0`.